### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by marcospereira**
Minor updates in preparation to 2.7.1.

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -275,7 +275,7 @@ object Dependencies {
     "com.github.ben-manes.caffeine" % "jcache"   % caffeineVersion
   ) ++ jcacheApi
 
-  val playWsStandaloneVersion = "2.0.1"
+  val playWsStandaloneVersion = "2.0.2"
```
Comment (renatocaval) [inline project/Dependencies.scala]: the build is failing because of this update. A regression in play-ws?

**Comment by marcospereira** [comment]
@renatocaval done.

**Comment by mkurz** [comment]
Fail: https://travis-ci.com/playframework/playframework/jobs/186433891#L2984-L2985

**Comment by marcospereira** [comment]
@mkurz we have a green build now.

**Comment by renatocaval** [comment]
Mergify triggered a new build, probably because it rebased the branch first. I will backport tomorrow morning. 

Time for automating backports with Mergify!

**Comment by renatocaval** [comment]
Backported on 2.7.x (b5ff799f4d0338d6c10e41b719d0c18d195c466e)

**Pull request merged**